src/fromager/commands/lint_requirements.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import sys
+
+import click
+from packaging.requirements import InvalidRequirement, Requirement
+
+from fromager import requirements_file
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument(
+    "input_files_path", nargs=-1, required=True, type=click.Path(exists=False)
+)
+def lint_requirements(input_files_path: list[click.Path]) -> None:
+    """
+    Command to lint the constraints.txt and requirements.txt files
+    This command takes a single wildcard path string for constraints.txt and requirements.txt.
+    It checks the formatting of these files and reports issues if found.
+    """
+
+    if len(input_files_path) == 0:
+        logger.error("no constraints.txt or requirements.txt found in given paths")
+        sys.exit(1)
+
+    flag = True
+
+    for path in input_files_path:
+        parsed_lines = requirements_file.parse_requirements_file(str(path))
```

Comment:
We need to convert this to string because the function `parse_requirements_file` only accepts `str | pathlib.Path`

Reply:
This is fine. For future reference, https://github.com/python-wheel-build/fromager/blob/main/src/fromager/clickext.py#L10 defines a class that can convert click.Path() to pathlib.Path(). We use that in a few other places.